src/test/java/org/alfresco/repo/coci/CheckOutCheckInServiceImplTest.java

```diff
@@ -618,15 +619,20 @@ public void testVersionAspectNotSetOnCheckIn()
         Map<QName, Serializable> bagOfProps = createTypePropertyBag();
         bagOfProps.put(ContentModel.PROP_CONTENT, new ContentData(null, MimetypeMap.MIMETYPE_TEXT_PLAIN, 0L, "UTF-8"));

```

Comment:
Is bagOfProps still required? Looks like we removed it. (See deleted lines below)